core/pom.xml

```diff
@@ -147,6 +147,51 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.0.1</version>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>../eclipse_plugin/lib</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>target</directory>
+                  <include>${project.artifactId}-${project.version}.jar</include>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
+        <executions>
+          <execution>
+            <id>copy-dependencies</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
```

Comment:
I'm seeing errors from the eclipse plugin on an initial build that go away on subsequent builds, presumably because the dependencies have been copied over. Is there a way for the plugin to declare its dependencies directly? If not, WDYT of using a script to set up the eclipse module's deps instead of doing it with maven?